specification/ai/data-plane/HealthInsights/preview/2023-09-01-preview/openapi.json

```diff
@@ -1405,7 +1405,7 @@
     },
     "ClinicalTrials": {
       "type": "object",
-      "description": "The clinical trials that the patient(s) should be matched to. \nThe trial selection can be given as a list of custom clinical trials and/or a list of filters to known clinical trial registries.\nIn case both are given, the resulting trial set is a union of the two sets.",
+      "description": "The clinical trials that the patient(s) should be matched to.\nThe trial selection can be given as a list of custom clinical trials and/or a list of filters to known clinical trial registries.\nIn case both are given, the resulting trial set is a union of the two sets.",
```

Comment:
What is causing this description change? If possible, it might be better to isolate the openapi changes into a separate PR, if the "enum to union" conversion shouldn't cause any openapi changes.